src/cheri/zylevels1.adoc

```diff
@@ -0,0 +1,171 @@
+
+<<zylevels1_lg_perm>> is a _refinement of_ <<c_perm>> and <<r_perm>>.
+That is, if either of the latter are clear, then <<zylevels1_lg_perm>> has no effect.
+The capability encoding might take advantage of this dependence for compression,
+and so clearing either <<c_perm>> or <<r_perm>> may result in
+<<zylevels1_lg_perm>> being reported as clear in <<GCPERM>>.
+
+[#zylevels1_gl_perm,reftext="GL(obal) Flag"]
+=== The Capability Global (GL) Flag
+
+The _Capability Global_ (GL) flag is a permission-like single-bit field which allows enforcing invariants on capability propagation in combination with the <<zylevels1_lg_perm>> and <<zylevels1_sl_perm>> bits described above.
+
+NOTE: For example, the software TCB may enforce that software has access to capabilities with <<zylevels1_sl_perm>> only to (subsets of) its runtime stack,
+and may ensure that all stack pointers lack <<zylevels1_gl_perm>>.
+In such a system, capabilties without <<zylevels1_gl_perm>>, including all those derived from the stack, are confined to registers and stack memory.
+Global capabilities, say, into heap memory, may be _attenuated_ to being local before being passed across a call;
+the callee will be unable to capture this pointer outside its stack.
++
+This specification defines only the architectural mechanics of this feature, for further information on how this can be used by software please refer to cite:[cheri-v9-spec].
+
+The _Capability Global_ flag holds one of two values:
+
+* 1: the capability is _global_.
+* 0: the capability is _local_.
+
+As with permissions, the _Capability Global_ flag of a valid capability can be cleared but never set (without reconstruction from a global superset capability).
+Unlike architectural permissions, the _Capability Global_ flag can be cleared even if the capability is sealed
+(both implicitly in loads from memory and explicitly with <<ACPERM>>).
+
+=== Interaction with <<infinite-cap,Infinite Capabilities>>
+
+As expected, <<infinite-cap,Infinite capabilities>> have both new permissions, <<zylevels1_lg_perm>> and <<zylevels_sl_perm>> set.
+Further, Infinite caps have <<zylevels1_gl_perm>> set to _global_.
+
+=== Interaction with <<ACPERM>> and <<GCPERM>>
+
+.Extended capability permissions bit field (see <<acperm_bit_field>>)
+[#zylevels1_acperm_bit_field]
+include::./img/zylevels1-acperm_bit_field.edn[]
+
+The <<zylevels1_gl_perm>>, <<zylevels1_sl_perm>>, and <<zylevels1_lg_perm>> fields
+are mapped into the <<acperm_bit_field,capability permissions bitfield>> (<<acperm_bit_field>>),
+used by <<ACPERM>> and <<GCPERM>>,
+as shown in <<zylevels1_acperm_bit_field>>.
+
+=== Interaction with <<LC>> and <<zalrsc_cheri>>
+
+As outlined above, Zylevels1 introduces two new constraints on capabilities loaded from memory,
+as part of a <<LC>> instruction `lc cd, offset(cs1)` or <<LR_C>> instruction `lr.c cd, offset(cs1)`.
+These may be phrased as "implicit <<ACPERM>>-s" performed on the loaded capability thus:
```

Comment:
I could see how referring to this as "implicit ACPERM" might confuse people. An ACPERM does legalisation and thus can also clear other permission bits than the ones named. However a capability load does not clear those, but only local-global related bits in certain cases.